README.md

```diff
@@ -44,8 +44,8 @@ May also function as reference Service Provider implementation.
 
 ### Generating a new key + self-signed cert
 
-    openssl req -newkey rsa:2048 -nodes -keyout config/server.key \
-      -x509 -out config/server.crt -config config/openssl.conf
+    openssl req -days 3650 -newkey rsa:2048 -nodes -keyout keys/saml_test_sp.key \
+      -x509 -out certs/sp/demo_sp.crt -config config/openssl.conf
```

Comment:
The key and the cert here are a matching pair so shouldn't they have the same name?

Also, I don't think we need an 'sp' folder since there is only one of them and it already has 'sp' in the name.

There are unrelated to this PR though I guess, so lgtm.